src/desfire/devices/pn532.py

```diff
@@ -40,13 +44,28 @@ def __init__(self, port: str, **kwargs):
         Keyword Args:
             All keyword arguments are passed to the pyserial.Serial constructor.
         """
-
         if not _has_serial:
             raise ImportError("pyserial is required for using PN532UARTDevice")
 
         self._uart = serial.Serial(port, **kwargs)
+        self._wakeup()
+        ic, ver, rev, support = self.firmware_version()
+        logger.info("PN532 initialized, found version {ver}.{rev}.")
         self._sam_configuration()
-        self._listen_for_passive_target(timeout=1)
+        self._listen_for_passive_target(timeout=0.2)
```

Comment:
Please do not change default values. What is the reason for you to change it? The better approach would be make this configurable by the user if that value needs to change